index.js

```diff
@@ -68,18 +92,26 @@ function Car() {
     - Besides the methods on Person.prototype, babies have the ability to `.play()`:
         + Should return a string "Playing with x", x being the favorite toy.
 */
-function Baby() {
+Baby.prototype = Object.create(Person.prototype);
+
+function Baby(name, age, favoriteToy) {
+  this.name = name;
+  this.age = age;
+  this.favoriteToy = favoriteToy;
+}
 
+Baby.prototype.play = function (){
+  return "Playing with" + this.favoriteToy;
 }
 
 /* 
   TASK 4
 
   In your own words explain the four principles for the "this" keyword below:
-  1. 
-  2. 
-  3. 
-  4. 
+  1. Default binding calls the standalone function
+  2. Implicit binding ties the this keyword to the object its infront of
+  3. Explicit binding ties the this keyword to a function outside of the function where it is being called
+  4. new binding the this is is applied to the new 
 */
```

Comment:
Default binding means the `this` keyword refers to the window/console
Implicit binding means the `this` keywords refers to the object that is to the left of the dot when calling a function is preceded by the .
Explicit binding occurs when the call or apply methods are used to refer to the function it is calling.
New binding means the `this` keyword refers to the specific instance of the object that is being created by a constructor function